### PR TITLE
Add Rupav Jain

### PR DESCRIPTION
**Pull request by rupav**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] ~~**Tier 2** ✏️: I wrote a post on DEV about a project I built.~~
  - [ ] ~~**Tier 3** ✨: I want my project to be highlighted on stream.~~
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by rupav** [comment]
*Image size is as mentioned of size, 544 x 544 px

File: _data/rupav/rupav.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Rupav Jain
+institution: Netaji Subhas Institute of Technology
+profile_pic: rupav.jpg
+quote: "Control is just an illusion. Servers will crash no matter what you do!"
```
Comment (ayan-b) [inline _data/rupav/rupav.md:5]: Please avoid using quotes. From README:
> quote: YOUR-SENIOR-QUOTE # no longer than 100 characters, avoid using quotes(") to guarantee the format remains the same.

Comment (rupav) [reply _data/rupav/rupav.md]: @ayan-b I have made the changes. Thanks for the review.

**Comment by rupav** [comment]
@ayan-b can you now approve it and merge it? It has been 2 days.

**Comment by ayan-b** [comment]
Don't worry, someone from GitHub education team will merge the PR very soon. :)

**Pull request merged**